src/redux/actions/index.js

```diff
@@ -34,3 +38,37 @@ export function setLanguageName(language) {
     payload: language,
   };
 }
+
+export function setUserPhrases(phrases) {
+  return {
+    type: SET_USER_PHRASES,
+    payload: phrases,
+  };
+}
+
+export function addNewPhrases(phrase) {
+  return async dispatch => {
+    const storedPhrases = await getData(NEW_KEY_PHRASES);
+    let dataToStore = null;
```

Comment:
This can be written more elegantly, possibly use a ternary
```suggestion
    const dataToStore = storedPhrases ? [...storedPhrases, phrase] : [phrase]
```

Reply:
Or any other solution